source/simulator/simulator_signals.cc

```diff
@@ -96,8 +96,25 @@ namespace aspect
 #define INSTANTIATE(dim) \
   template struct SimulatorSignals<dim>;
 
-
   ASPECT_INSTANTIATE(INSTANTIATE)
 
+  // Now also instantiate static member variables:
+  template
```

Comment:
Can you use the ASPECT_INSTANTIATE macro above? This would do the ``dim`` part for you.